Form-Controls/index.html

```diff
@@ -1,28 +1,114 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="css/styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <h1>Product Pick</h1>
+      </header>
+      <main>
+        <form action="resilts.html" method="get">
+          <div class="grid--1">
+            <label for="name">Name</label>
+            <input placeholder="First Name" name="name" id="name" required />
+          </div>
+          <div class="grid--1">
+            <label for="email">Email</label>
+            <input
+              placeholder="Email address
+              "
+              type="email"
+              name="Email"
+              id="email"
+              required
+            />
+          </div>
+          <div class="grid--1">
```

Comment:
I would be looking for a fieldset here. Look into what fieldset does and why it' more useful than a div.

Div and span are semantically null in HTML.

Reply:
@SallyMcGrath 😊

Thanks again for your helpful suggestions! I’ve implemented a <fieldset> for the form and learned how to use the <legend> effectively. Your guidance has really improved my understanding!

I appreciate your support🙏